content/docs/5.1/deploy/operate/upgrade.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Upgrade
+description: Learn how to upgrade TiKV using TiUP
+menu:
+    "5.1":
+        parent: Operate TiKV
+        weight: 1
+---
+
+This document is targeted for users who want to upgrade from TiKV 4.0 versions to TiKV 5.0 versions, or from TiKV 5.0 to a later version.
+
+## Preparations
+
+This section introduces the preparation works needed before upgrading your TiKV cluster, including upgrading TiUP and the TiUP Cluster component.
+
+### Step 1: Upgrade TiUP
+
+Before upgrading your TiKV cluster, you first need to upgrade TiUP.
```

Comment:
why do the users need to upgrade tiup firstly before upgrading a tikv cluster?

Reply:
TiKV's new feature may not be supported by the old version TiUP.

Reply:
It's recommended by TiUP experts.